test/lib/usd/utils/test_DiffValuesHalfVecs.cpp

```diff
@@ -0,0 +1,372 @@
+#include <mayaUsdUtils/DiffPrims.h>
+
+#include <pxr/base/gf/vec2d.h>
+#include <pxr/base/gf/vec2f.h>
+#include <pxr/base/gf/vec2h.h>
+#include <pxr/base/gf/vec2i.h>
+#include <pxr/base/gf/vec3d.h>
+#include <pxr/base/gf/vec3f.h>
+#include <pxr/base/gf/vec3h.h>
+#include <pxr/base/gf/vec3i.h>
+#include <pxr/base/gf/vec4d.h>
+#include <pxr/base/gf/vec4f.h>
+#include <pxr/base/gf/vec4h.h>
+#include <pxr/base/gf/vec4i.h>
+#include <pxr/base/vt/array.h>
+#include <pxr/base/vt/value.h>
+
+#include <gtest/gtest.h>
+
+using namespace PXR_NS;
+using namespace MayaUsdUtils;
+
+//----------------------------------------------------------------------------------------------------------------------
+TEST(DiffValuesHalfVecs, vec2hCompareValuesSame)
+{
+    VtValue    baselineValue(GfVec2h(1, 12));
```

Comment:
Probably doesn't matter at all, but I was surprised to see that in the half float vector test the only test values you have are integers :)